sdk/servicebus/service-bus/test/internal/unit/batchingReceiver.spec.ts

```diff
@@ -814,7 +818,4 @@ function assertListenersRemoved(rheaReceiver: RheaPromiseReceiver): void {
       `No listeners should be registered for ${eventName} on the receiver.session`
     );
   }
-
-  // check the session as well
-  rheaReceiver.session;
```

Comment:
Was this missing an assert?

Reply:
We already check `rheaReceiver.session.listeners` above. is there anything else to check?